### PR TITLE
Add content/es/docs/reference/glossary/service.md

### PR DESCRIPTION
**Pull request by daverick**
add the glossary/service for the ES translation

### CODE REVIEW

**Comment by daverick** [comment]
@glo-pena
arreglado

**Comment by sftim** [comment]
I am concerned that this translation does not show the reader the name that the API uses for this object (ie, `Service`).

How do localizations typically address this?

**Comment by glo-pena** [comment]
We do not translate names that are Kubernetes specific e.g.- kubelet.
For those that do have a more generic sense (service, node) we do tend to translate in the overall text and make it explicit when it does reference an API call or object, in which case it will be presented to the reader also in between `backticks`.

**Comment by glo-pena** [comment]
Perhaps for clarity in the Glossary we should stick to the original in the header and translate in parenthesis...

#Service (servicio)

**Comment by daverick** [comment]
Hi @glo-pena,

I have updated mi PR to take into account your request to not translate `Service` as it is a API object.
Is there anything else I should change?

File: content/es/docs/reference/glossary/service.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Service (servicio)
```
Comment (raelga) [inline content/es/docs/reference/glossary/service.md]:
```suggestion
title: Service
```

File: content/es/docs/reference/glossary/service.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+id: service
+date: 2018-04-12
+full_link: /docs/concepts/services-networking/service/
+short_description: >
+  Un objeto de la API de Kubernetes que describe como se accede a las aplicaciones, tal como un conjunto de Pods, y que puede describir puertos y balenceadores de carga.
```
Comment (raelga) [inline content/es/docs/reference/glossary/service.md]:
```suggestion
  Un Service, servicio en castellano, es el objeto de la API de Kubernetes que describe cómo se accede a las aplicaciones, tal como un conjunto de Pods, y que puede describir puertos y balanceadores de carga.
```

File: content/es/docs/reference/glossary/service.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+aka:
+tags:
+- fundamental
+- core-object
+---
+ Un objeto de la API de Kubernetes que describe como se accede a las aplicaciones, tal como un conjunto de {{< glossary_tooltip text="Pods" term_id="pod" >}}, y que puede describir puertos y balenceadores de carga.
```
Comment (raelga) [inline content/es/docs/reference/glossary/service.md]:
```suggestion
 Un Service, servicio en castellano, es el objeto de la API de Kubernetes que describe cómo se accede a las aplicaciones, tal como un conjunto de {{< glossary_tooltip text="Pods" term_id="pod" >}}, y que puede describir puertos y balanceadores de carga.
```

**Comment by raelga** [comment]
Tracked at https://github.com/kubernetes/website/issues/13948

**Comment by raelga** [comment]
Thank you very much @daverick!

Can you squash your commits into a single one?

**Comment by raelga** [comment]
Muchas gracias @daverick !

/lgtm

Solo faltaría que @glo-pena confime que su review está resuelta y lo podemos publicar! 🎉

**Comment by raelga** [comment]
Gracias @daverick and @glo-pena !

:blue_heart: 

/approve

**Pull request merged**